taxonomy-editor-frontend/src/pages/project/search/SearchExpressionInput.tsx

```diff
@@ -0,0 +1,50 @@
+import { FormControl, OutlinedInput, InputAdornment } from "@mui/material";
+import SearchIcon from "@mui/icons-material/Search";
```

Comment:
nit: Named imports are more maintainable imo (https://stackoverflow.com/a/58103685) and does not have any impact on performance with `Vite`'s tree-shaking (https://mui.com/material-ui/guides/minimizing-bundle-size/#when-and-how-to-use-tree-shaking)
```suggestion
import { SearchIcon } from "@mui/icons-material";
```

Reply:
even if mui recommends the opposite ? (https://mui.com/material-ui/icons/#usage)